modules/base/include/modules/base/processors/datasource.h

```diff
@@ -96,7 +96,7 @@ class DataSource : public Processor {
      * @param content A content type passed into the FileProperty, usually 'volume', 'image' etc.
      * @see FileProperty
      */
-    DataSource(InviwoApplication* app = InviwoApplication::getPtr(), const std::string& file = "",
+    DataSource(InviwoApplication* app = util::getInviwoApplication(), const std::string& file = "",
```

Comment:
remove include?

Reply:
not possible. Used in line 130 to access the DataReaderFactory
https://github.com/inviwo/inviwo/blob/be14babe4453b7a7765ea8c5a2881a0fb3b5bafa/modules/base/include/modules/base/processors/datasource.h#L130